### PR TITLE
New Bluespace Productions Products!

### PR DESCRIPTION
**Pull request by AyyRobotics**
## About The Pull Request

Adds the mining encampment capsule, the emergency medical capsule, the security barrier capsule XL,  the *party capsule*, and the space shelter capsule, crates to buy them in, some sprites for them, some notes, and the "BSP Collapsable Shuttle."

Also adds "Bluespace Productions" as a Nanotrasen Subsidiary, only shows up in the notes.

Encampment: A 14x16 base with the amenities of standard pods, but with a custom food vendor and equipment vendor
image

Emergency Medical: a 7x5 capsule made for situations when medical can't keep up, contains a sleeper, stasis bed (same sprite), and a nanomed.
image

Security Barrier: a 3x3 glass barricade for security use with laser weapons. Perfect against nukies. Windoors are public.
image

Space Shelter: a (technically) 9x9 shelter with most of the amenities of a standard shelter capsule, but more spacey.
image

Party Capsule: a 7x7 party area for when the station is in the mood to celebrate. Comes in the Party Crate.
image

BSP Collapsable Shuttle: a small, $2500 emergency shuttle with the theme of a bluespace shelter, contains seating for around ~20 and has two sleepers, wildlife turrets at the front (space carp, I guess?), and the most important, a CAS deck in the maint area.
image

## Why It's Good For The Game

Encampment:
The mining shelter capsules never really seemed like a "base" in the sense that the aux does. This remedies that by having a not-cheap not-expensiveish capsule that fits that role. Luxury and Bar aren't real bases either, and unlike them, this one focuses more on utility rather then luxury.

Emergency Medical:
We've all had those times in which medical gets overwhelmed and doesn't have enough sleepers/doctors to take care of a situation, or when medical is too far away from a threat (like blob). This helps that by providing a decently-sized, decently-stocked medical area for those doctors on the front line.

Security Barrier: Lets face it, barrier grenades are boring. Place one single barricade down and that's it. Somewhat effective but not used often. This helps that by allowing sec to deploy a hallway-sized barricade that they are able to shoot through. Combined with a riot officer or a wall, this barricade could likely hold down areas quite effectively from many threats.

Space Shelter: This one is two-fold, as it can both provide a great starting place for people to build things off-station, but can be useful in a pinch when space exploring or getting stuck floating in space.

Party: I mean, partying. Can't go wrong there can you?

BSP Collapsable Shuttle: Cheap, looks good, on the smaller side but good if there isn't going to be many people escaping that day. Nothing really special here, just another shuttle variant and those are always nice.

## Changelog
:cl:
add: Added 5 new bluespace capsules
add: Added a new shuttle
add: Adds Bluespace Productions as a Nanotrasen subsidiary
imageadd: added new capsule icons
balance: Party Crate costs 2500 due to the addition of a capsule.
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
Due to all of these blooody reinforced window buffs, maybe make the barrier capsule use normal windows

**Comment by AyyRobotics** [comment]
Wait hold on fuck I put 2 comms consoles in the shuttle by accident

**Comment by AyyRobotics** [comment]
Alright fixed it

**Comment by AyyRobotics** [comment]
Damn you Travis

**Comment by AyyRobotics** [comment]
Okay I think I fixed what got messed up, is there a way to like, restart travis or something

**Comment by PowerfulBacon** [comment]
How does spawning multiple shuttles capsules work?

**Comment by AyyRobotics** [comment]
The shuttle isn't a capsule, it's just capsule themed

**Comment by AyyRobotics** [comment]
Alright im working on fixing the errors

**Comment by AyyRobotics** [comment]
Oh thank fuck

**Comment by BettonnCZ** [comment]
+1 cause its cool and good

**Review by PowerfulBacon — COMMENTED** [review]
I see a lot of tiny fans in the middle of rooms rather than in the doors

**Comment by PowerfulBacon** [comment]
Also people will be very unhappy when they buy a 'shuttle' capsule and it cant fly

**Comment by PowerfulBacon** [comment]
Making a capsule with the actual emergency shuttle area will probably break things

File: _maps/shuttles/emergency_shelter.dmm [hunk 1]
```diff
@@ -0,0 +1,698 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"aa" = (
+/turf/template_noop,
+/area/template_noop)
+"ab" = (
+/obj/effect/spawner/structure/window/plastitanium,
+/turf/open/space/basic,
+/area/shuttle/escape)
```
Comment (PowerfulBacon) [inline _maps/shuttles/emergency_shelter.dmm:8]: Why is this area the escape shuttle?

Comment (AyyRobotics) [reply _maps/shuttles/emergency_shelter.dmm:8]: Oh, I think that's meant to be a passthrough, hold on

Comment (AyyRobotics) [reply _maps/shuttles/emergency_shelter.dmm:8]: Oh the windows, hold on

**Comment by AnjelicaChaykovsky** [comment]
You based motherfucker, i recently made a post about how cool  it would be to make more use of bluespace capsules and  it actually exists

**Comment by AyyRobotics** [comment]
> You based motherfucker, i recently made a post about how cool  it would be to make more use of bluespace capsules and  it actually exists

Well ain't that convenient? There is indeed unused potential when it comes to these

**Comment by AyyRobotics** [comment]
Fuck I forgot that that map had the old filepath, hold on

Okay fixed it

**Comment by AnjelicaChaykovsky** [comment]
So you didnt see my forums post where  i talked about capsules around two weeks ago? 
Damn thats a shame  i wish i could have a chat with you in that post.

**Comment by AyyRobotics** [comment]
> So you didnt see my forums post where  i talked about capsules around two weeks ago?
> Damn thats a shame  i wish i could have a chat with you in that post.

Yeah, didn't end up seeing it, bet there were some good ideas there. At least it's not too difficult to make more!

**Comment by AyyRobotics** [comment]
Forgot that Tweak is the wrong word for it, changed it to balance

**Comment by AnjelicaChaykovsky** [comment]
Is this stuff going to be crate only?  
Or  is it going to be in vendors too? 
Also what is the process for adding new room to  capsule?

**Comment by AyyRobotics** [comment]
> Is this stuff going to be crate only?
> Or  is it going to be in vendors too?
> Also what is the process for adding new room to  capsule?

4 of them are crate exclusive, the barricade, party, medical, and space capsules are either given their own crates or, in the case of the party crate, is included there. The mining camp capsule is gonna be in the vendor though.
To make a new room, I just make a new .dmm file in /templates, go to "shelters.dm" and put the code there, which is as easy as copying one and changing the values needed. For the item itself, I just go to "survival_pod.dm" and copy the code from another one, and replace the values needed.
Also, if your adding your own, don't worry about the "shelter_id" box in shelters.dm, just put whatever you want to call the shelter id in it. For instance, the party crate uses "shelter_theta"

File: code/game/objects/structures/signs/signs_departments.dm [hunk 2]
```diff
@@ -73,6 +73,10 @@
 /obj/structure/sign/departments/medbay/alt
 	icon_state = "bluecross2"
 
+/obj/structure/sign/departments/medbay/alt2
+	icon_state = "medicalsign"
+	desc = "An image of the Caduceus, with a small cross. Makes no sense to someone not versed in human mythology. You'll probably get help here."
```
Comment (Dingo-Dongler) [inline code/game/objects/structures/signs/signs_departments.dm]: The rod of Asclepius is the medical one. Just change it to "Makes no sense to someone ~~not~~ versed in human mythology."

**Comment by AyyRobotics** [comment]
Oh come on

**Comment by AyyRobotics** [comment]
After looking into this bug it seems it's byond-based, and as such I both have no idea how to solve it and have no idea if it will affect merging.

**Comment by PowerfulBacon** [comment]
Inconsistent areas still, some are emergency shuttle for some reason, some are NOOP and some are survival pods.

**Comment by AyyRobotics** [comment]
As-in, not the problem with my code

**Comment by AyyRobotics** [comment]
> Inconsistent areas still, some are emergency shuttle for some reason, some are NOOP and some are survival pods.

Oh shit, a'ight I'll handle this, give me a bit

File: code/game/objects/structures/signs/signs_departments.dm (hunk 2 above)
Comment (AyyRobotics) [reply code/game/objects/structures/signs/signs_departments.dm]: I originally thought that aswell, but it does seem that the Caudecus is also used as a medical symbol, even when the Rod of Asclepius is the more accurate one. But yeah the text could use an update, I'll get on that

**Comment by AyyRobotics** [comment]
> Inconsistent areas still, some are emergency shuttle for some reason, some are NOOP and some are survival pods.
> 
> Oh shit, a'ight I'll handle this, give me a bit

Alright after looking, I don't see any inconsistencies. All the noops are in the correct places, the only use of the escape shuttle area is for the escape shuttle, and all the survival pod ones are in their correct places so I dunno where you see them. If you could point out the specific ones are that'd be great, thanks

**Comment by PowerfulBacon** [comment]
Oh, I see you added another shuttle too

Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)

**Comment by AyyRobotics** [comment]
> Oh, I see you added another shuttle too
> Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)

I thought I revised them for that, lemme check

**Comment by AyyRobotics** [comment]
> Oh, I see you added another shuttle too
> Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)
> 
> I thought I revised them for that, lemme check

Alright I *did* revise them for that, though I missed a few spots. Gonna update that right quick. Pictures are a bit outdated

**Comment by PowerfulBacon** [comment]
> > Oh, I see you added another shuttle too
> > Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)
> > I thought I revised them for that, lemme check
> 
> Alright I _did_ revise them for that, though I missed a few spots. Gonna update that right quick. Pictures are a bit outdated

Oh, I apologise then, I've been looking at the pictures. I'll have a long in fastdmm2 at some point.

**Comment by AyyRobotics** [comment]
> Oh, I see you added another shuttle too
> Also, tiny fans only work by blocking atmos, having them in the middle of the rooms doesn't do anything, they need to be in doorways (unless its for deco, but having the survival pods not have their air rush out is cool)
> I thought I revised them for that, lemme check
> 
> Alright I did revise them for that, though I missed a few spots. Gonna update that right quick. Pictures are a bit outdated
> 
> Oh, I apologise then, I've been looking at the pictures. I'll have a long in fastdmm2 at some point.

All good, did fix what needed fixing though, so thanks for pointing that out

**Comment by AyyRobotics** [comment]
Oh also that bug I had sorted itself out. Neat.

**Review by zeskorion — CHANGES_REQUESTED** [review]
Not sure about the sec one using reinforced glass, especially the fucking hollow windows that take ages to deconstruct

Remove the basalt from the encampment, because it makes no sense if spawned in space

**Comment by AyyRobotics** [comment]
> Not sure about the sec one using reinforced glass, especially the fucking hollow windows that take ages to deconstruct
> Remove the basalt from the encampment, because it makes no sense if spawned in space

I considered that when making it, however it would just leave the courtyard full of rock when it was spawned, so I decided against it so it would both be easier to access and just look better.

**Review by zeskorion — APPROVED** [review]

**Pull request merged**